src/lightning_lite/plugins/environments/slurm_environment.py

```diff
@@ -134,7 +128,17 @@ def local_rank(self) -> int:
     def node_rank(self) -> int:
         return int(os.environ["SLURM_NODEID"])
 
-    def resolve_root_node_address(self, root_node: str) -> str:
+    @staticmethod
+    def resolve_root_node_address(nodes: str) -> str:
+        """The node selection format in SLURM supports several formats.
```

Comment:
Adding this for clarity, for future code readers, to make sure what is supported. If you look on the internet for SLURM documentation regarding this, good luck lol.